transformations/src/main/java/jp/wasabeef/glide/transformations/BlurTransformation.java

```diff
@@ -102,6 +103,8 @@ public BlurTransformation(Context context, int radius, int sampling) {
     blur.forEach(output);
     output.copyTo(bitmap);
 
+    source.recycle();
+    resource.recycle();
```

Comment:
Maybe we can use ApplicationContext, `mContext = context.getApplicationContext()`. @wasabeef